Print_List_of_Odd_Numbers.py

```diff
@@ -18,6 +18,9 @@ def print_error_messages():
   if(n==0):
     start+=1
     print(start)
-  else:
-    if(n%2!=0):
-    print(start)
+    patch-2
+  elif(n%2!=0):
+    else:
+      if(n%2!=0):
+        master
```

Comment:
master is not defined anywhere else.
I must say.
So, it will raise error. Because it is not mapped/declared/initiated.